changelog

```diff
@@ -0,0 +1,17 @@
+PROJECT: https://github.com/faf0/sct
+
+1.3: zvezdochiot on 02 Jan 2018
+* Rename utilites for X11
+* Add -h option to print usage information
+* Add license documentation
+* Add changelog
```

Comment:
please rename this file to `CHANGELOG` ( no need to update this line)

Reply:
I agree.